MinPlatformPkg/Test/Library/TestPointCheckLib/MmCheckPaging.c

```diff
@@ -1,16 +1,17 @@
 /** @file
```

Comment:
This is another file with a lot of whitespace only changes. Please remove them to minimize deltas to the upstream.

Reply:
Fixed.